libraries/AP_RangeFinder/AP_RangeFinder_BBB_PRU.cpp

```diff
@@ -67,16 +78,16 @@ bool AP_RangeFinder_BBB_PRU::detect()
     FILE *file = fopen("/lib/firmware/rangefinderprutext.bin", "rb");
     if(file == nullptr)
     {
-        result = false;
+        goto fail_fopen;
     }
-
-    if(fread(ram, PRU0_IRAM_SIZE, 1, file) != 1)
+    
```

Comment:
Extraneous whitespace; see if you can turn on "show trailing whitespace" in your editor.